src/components/AddPaymentMethodMenu.js

```diff
@@ -72,7 +73,7 @@ export default compose(
     withWindowDimensions,
     withLocalize,
     withOnyx({
-        payPalMeUsername: {
+        payPalMeUserDetails: {
             key: ONYXKEYS.NVP_PAYPAL_ME_ADDRESS,
```

Comment:
I think we should rename this Onyx key because it now contains more than just the PayPal username

Reply:
What do plan do you have in mind to rename this key without breaking the app?

I am thinking something like:
1. In the API, if `$massageData` is true, return a new key `NVP_PAYPAL` instead of `NVP_PAYPAL_ME_ADDRESS`
2. In this PR use the new Onyx key
3. When both are deployed, in the clean up PR remove the old Onyx key.

Reply:
Which means I need to make a new API PR to add NVP_PAYPAL